client-app/shared/checkout/components/checkout-labeled-block.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <p class="font-extrabold text-base mb-1">{{ label }}</p>
```

Comment:
Why P tag is used here? Maybe better to use plain div.

Reply:
@Flanker72 I think P tag is more semantic, since P is used for paragraphs of text and DIV is better referred for a block of content having different nested elements